src/loaders/ObjectLoader.d.ts

```diff
@@ -31,4 +31,6 @@ export class ObjectLoader extends Loader {
 		animations: AnimationClip[]
 	): T;
 
+	loadAsync<ObjectType extends Object3D>( url: string, onProgress?: ( event: ProgressEvent ) => void ): Promise<ObjectType>;
```

Comment:
Should this be like this?

```js
loadAsync<T extends Object3D>( url: string, onProgress?: ( event: ProgressEvent ) => void ): Promise<T>;
```

Reply:
I took type declaration name from `load`. Should I change `ObjectType` to `T` there as well?

https://github.com/mrdoob/three.js/pull/21001/files/5b89174f415cd527f9e486a4b5ea4f085a61a05f#diff-bca29ae4504d02730174f610a8aba5e3772046ee1734e1cb5c49f2b85e024007R14

Reply:
No idea to be honest. I don't know much about types...
We can merged it as it is and let other people fix it if it's wrong.